openshift-install-powervs

```diff
@@ -1287,29 +1372,23 @@ function main {
     "-var")
       shift
       var="$1"
-      #TODO: Need validation on variable key=value, currently it is only checking if equals sign is present or not
-      [[ "$var" != *"="* ]] && error "The given -var option must be a variable name and value separated by an equals sign, eg: -var \"key=value\""
+      [[ $(echo "$var" | grep -o "[[:blank:]]" | wc -l) -gt 0 ]] && error "The -var '$var' option cannot have whitespaces in it. Please use -var-file instead."
```

Comment:
Move these checks to `process_var` and please take care of sensitive data. Just show the variable name in the error log.

Reply:
Resolved